### PR TITLE
Remove the MobileCLIP and use the new base image

### PR DESCRIPTION
**Pull request by wanliAlex**
* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)
bug fix

* **What is the current behavior?** (You can also link to an open issue here)
We are using base image 29 and supporting mobileclips

* **What is the new behavior (if this is a feature change)?**
We change to base image 30, with openclip downgraded back to 2.24.0.
Mobileclip models are also removed from the model registry.

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
no

* **Have unit tests been run against this PR?** (Has there also been any additional testing?)

* **Related Python client changes** (link commit/PR here)

* **Related documentation changes** (link commit/PR here)

* **Other information**:

* **Please check if the PR fulfills these requirements**
- [ ] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes / features)

### CODE REVIEW

**Comment by wanliAlex** [comment]
Close to avoid unnecessary gh runs

**Review by farshidz — DISMISSED** [review]

**Comment by wanliAlex** [comment]
Close to avoid GH runs

File: src/marqo/s2_inference/model_registry.py [hunk 1]
```diff
@@ -364,6 +364,8 @@ def _get_open_clip_properties() -> Dict:
 
         # New models as of Marqo 2.7.0
         # Added for: Open CLIP 2.24.0
+        # TODO the open_clip/ViT-SO400M-14-SigLIP-384/webli is not available on torch 1.12
+        # Add this model back when we upgrade the torch version
```
Comment (papa99do) [inline src/marqo/s2_inference/model_registry.py]: So we kept this model, right? The comment's a bit confusing

**Review by papa99do — COMMENTED** [review]
The description `Mobileclip models are also not removed from the model registry` is conflict with the title :)

File: src/marqo/s2_inference/model_registry.py (hunk 1 above)
Comment (wanliAlex) [reply src/marqo/s2_inference/model_registry.py]: Updated

**Review by papa99do — APPROVED** [review]
LGTM

**Pull request merged**